src/Syntax.php

```diff
@@ -11,6 +11,7 @@
     private $closingTagMarker;
     private $parameterValueSeparator;
     private $parameterValueDelimiter;
+    private static $regexCache = array();
```

Comment:
No `static` values please. Memoization is a nice idea but don't force it to other instances.